packages/perseus/src/widgets/radio/choice.new.tsx

```diff
@@ -1,273 +1,50 @@
-        color: color.offBlack,
-    },
-
-    choicesContainer: {
-        display: "flex",
-    },
-
-    optionContainer: {
-        flex: 1,
-        color: color.offBlack,
-        userSelect: "text",
-    },
-
-    choiceRow: {
-        display: "flex",
-        flexDirection: "row",
-        justifyContent: "flex-start",
-        alignItems: "center",
-        paddingBlock: spacing.xSmall_8,
-        paddingInlineStart: spacing.xSmall_8,
-        position: "relative",
-        minWidth: "100%",
-        insetInlineStart: 0,
-        // The height and background will ensure content will be scrolled
-        // behind the ChoiceIcon and will not be visible
-        height: "100%",
-        background: color.white64, // TODO: LEMS-3108 address light/dark mode theme
-    },
-
-    choiceIcon: {
-        flexGrow: 0,
-        flexShrink: 0,
-        flexBasis: "30px",
-        position: "sticky",
-        insetInlineStart: 0,
-        background: "linear-gradient(to right, white 70%, transparent)",
-        // TODO: LEMS-3107 not properly working fix this by moving in a css file
-        ":dir(rtl)": {
-            background: "linear-gradient(to left, white 70%, transparent)",
-        },
-        paddingInlineStart: spacing.xSmall_8,
-        paddingInlineEnd: spacing.xSmall_8,
-        paddingTop: spacing.xxxSmall_4,
-        zIndex: 1,
-        alignItems: "center",
-        justifyContent: "center",
-        height: "100%",
-    },
-
-    choiceContent: {
-        paddingInlineStart: spacing.small_12,
-        textAlign: "start",
-        flex: 1,
-        paddingTop: spacing.xxxSmall_4,
-        minWidth: 0,
-    },
-
-    rationale: {
-        display: "block",
-        padding: intermediateCheckboxPadding,
-        paddingTop: 0,
-        marginInlineStart: 54,
-        color: color.offBlack64,
-        [mediaQueries.smOrSmaller]: {
-            padding: intermediateCheckboxPaddingPhone,
-            paddingTop: 0,
-        },
-    },
-} as const;
+const Choice = (props: IndicatorProps) => {
+    const showCorrectness = props.showCorrectness;
+    const buttonRef = useRef<HTMLButtonElement>(null);
+    const indicatorShape = props.isMultiSelect ? "square" : "circle";
+    const clickHandler = showCorrectness
+        ? undefined
+        : () => {
+              buttonRef.current?.click();
+          };
+    const classes = [styles.choice]
+        .concat(showCorrectness ? [styles["is-" + showCorrectness]] : [])
+        .join(" ");
+    return (
+        /*
+            A click handler is added to the <li> element to accommodate mouse users.
+            Keyboard users will still be able to use the <button> element inside
+                the <Indicator>.
+            Therefore, WCAG 2.1.1 is still satisfied since functionality is
+                aligned with the input method.
+         */
+        // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-noninteractive-element-interactions
```

Comment:
This is another great comment that will save us future confusion, while also allowing us to quickly reassess as WCAG requirements evolve in the future